recipes/source.rb

```diff
@@ -1,47 +1,45 @@
 include_recipe 'build-essential'
 
-Array(node[:ruby_installer][:source_package_dependencies]).each do |pkg|
+Array(node['ruby_installer']['source_package_dependencies']).each do |pkg|
   package pkg
 end
 
-bash "install_ruby" do
-  cwd "/usr/src"
+bash 'install_ruby' do
```

Comment:
The `ark` community cookbook is really handy, and I would use it unless there is a good reason not to. That said, I appreciate limiting the scope of these PRs and wouldn't blame you for pushing it to later.